test/000_fullchain.test.ts

```diff
@@ -99,7 +105,109 @@ describe('IexecPoco (IT)', function () {
         };
     }
 
-    describe('MatchOrders', function () {
+    it('[1] Sponsorship, beneficiary, callback, BoT, replication', async function () {
+        const volume = 3;
+        // Create deal.
+        const orders = buildOrders({
+            assets: ordersAssets,
+            prices: ordersPrices,
+            requester: requester.address,
+            tag: standardDealTag,
+            beneficiary: beneficiary.address,
+            callback: callbackAddress,
+            volume,
+            trust: 1, // TODO use 5 workers.
+        });
+        const { dealId, dealPrice, schedulerStakePerDeal } =
+            await iexecWrapper.signAndSponsorMatchOrders(...orders.toArray());
+        const taskPrice = appPrice + datasetPrice + workerpoolPrice;
+        const schedulerStakePerTask = schedulerStakePerDeal / volume;
+        const workerRewardPerTask = await iexecWrapper.computeWorkerRewardPerTask(
+            dealId,
+            PocoMode.CLASSIC,
+        );
+        const schedulerRewardPerTask = workerpoolPrice - workerRewardPerTask;
+        // Check initial balances.
+        await checkBalancesAndFrozens({
```

Comment:
maybe for the future:
`const before = await getBalancesAndFrozens({..`
then in `for` loop `checkBalancesAndFrozens` with `before` involved

Reply:
Added a TODO for that.